__tests__/app.test.js

```diff
@@ -107,4 +107,51 @@ describe("GET /api/reviews/review_id", () => {
         expect(body.message).toBe("No review found for review_id: 1000");
       });
   });
-});
\ No newline at end of file
+});
+
+describe.only("GET /api/reviews/:review_id/comments", () => {
+  test("GET: 200 - responds with a 200 status and an array of comments which match the input review_id", () => {
+    return request(app)
+      .get("/api/reviews/2/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        expect(comments).toHaveLength(3);
+        comments.forEach((comment) => {
+          expect(comment).toMatchObject({
+            comment_id: expect.any(Number),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            review_id: 2,
+          });
+        });
+      });
+  });
+  test("GET: 200 - responds with a 200 status with array ordered by created_at from most recent comment", () => {
+    return request(app)
+      .get("/api/reviews/2/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        expect(comments).toBeSortedBy("created_at", { ascending: true });
+      });
```

Comment:
Just missing one happy path here - what happens if the id is valid but has no associated comments?